docs/sources/contribute/documentation.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Documentation"
+description = "Contributing to documentation"
+type = "docs"
+[menu.docs]
+parent = "contribute"
+weight = 2
++++
+
+# Contributing to documentation
+
+## How do I contribute?
+
+If you’re unsure about where to start, check out some of our [open docs issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3Atype%2Fdocs).
+
+Sometimes it can be difficult to understand an issue when you're just getting started. We strive to keep a collection of [beginner-friendly issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3Atype%2Fdocs+label%3A"beginner+friendly") that is more suitable for first-time contributors.
+
+When you’ve found an issue you want to work on, you’re encouraged to comment on the issue to let other people know you intend to work on it.
+
+## Community
+
+If you have questions on a specific issue, post a comment to ask for clarification, or to give feedback.
+
+For general discussions on documentation, you’re welcome to join the `#docs` channel on our [public Grafana Slack](http://slack.raintank.io) team.
+
+## Guidelines
+
+All Grafana documentation is written using [Markdown](https://en.wikipedia.org/wiki/Markdown), and can be found in the [docs](https://github.com/grafana/grafana/tree/master/docs) directory in the [Grafana GitHub repository](https://github.com/grafana/grafana). The [documentation website](https://docs.grafana.com.)  is generated with [hugo](https://gohugo.io) which uses [Blackfriday](https://github.com/russross/blackfriday) as its Markdown rendering engine.
```

Comment:
Link to documentation should be `https://grafana.com/docs` and not https://docs.grafana.com.